server/routes/serviceProviderReferrals/interventionProgressPresenter.ts

```diff
@@ -0,0 +1,23 @@
+import { ActionPlan, SentReferral, ServiceCategory } from '../../services/interventionsService'
+import utils from '../../utils/utils'
+
+export default class InterventionProgressPresenter {
+  constructor(
+    private readonly referral: SentReferral,
+    private readonly serviceCategory: ServiceCategory,
+    private readonly actionPlan: ActionPlan | null
+  ) {}
+
+  readonly createActionPlanFormAction = `/service-provider/referrals/${this.referral.id}/action-plan`
+
+  readonly text = {
+    title: utils.convertToTitleCase(this.serviceCategory.name),
+    actionPlanStatus: this.actionPlanSubmitted ? 'Submitted' : 'Not submitted',
+  }
+
+  readonly actionPlanStatusColour: 'active' | 'inactive' = this.actionPlanSubmitted ? 'active' : 'inactive'
```

Comment:
these don't sound like colours :P

Reply:
Good point - gonna rename it to "style".